packages/hash/external-services/kratos/kratos.dev.yml

```diff
@@ -78,3 +97,8 @@ identity:
   schemas:
     - id: default
       url: file:///etc/config/kratos/identity.schema.json
+
+courier:
+  template_override_path: /etc/config/kratos/templates
+  smtp:
+    connection_uri: set-through-env-arg-COURIER_SMTP_CONNECTION_URI
```

Comment:
Are you able to set this dummy value because the environment variable overrides whatever we specify in the config?

Reply:
Yes, exactly, I think for dev mode, we should maybe just let it be the mailslurper url and add a comment as we do for `SELFSERVICE_METHODS_LINK_CONFIG_BASE_URL` above. I will do this now

Reply:
38b3db19ea3dcf1c824b38b19bc3b50d32ea0b03